pypeclub/OpenPype
# General: Run applications as separate processes under linux

## Brief description
Launched application on linux are child processes of OpenPype which may cause a huge issues of hanging processes, closing all children processes etc.

## Description
The goal is to create detached process on linux with creating mid-process that will launch the process as background process and exits. The mid process is new executable in OpenPype called `linux_app_launcher` which should work from code too.

The mid executable gives ability to set environment variables and detach the process.

## Notes
- it should be possible to detach the process in much easier way but I didn't find any that actually works

## How to test
1. Run application on linux
2. Check if the launched application is children process of OpenPype (using `pstree`)
